particles/Tutorial/Kotlin/Blackjack/Dealer.kt

```diff
@@ -0,0 +1,41 @@
+package arcs.tutorials.blackjack
+
+import arcs.sdk.Handle
+import arcs.sdk.Utils.log
+
+class Dealer : AbstractDealer() {
+    val name = "Dealer"
+
+    override fun getTemplate(slotName: String) = """
+        ${name}: <button type="button" on-click="onHit"> Hit </button> 
+        <button type="button" on-click="onStand"> Stand </button>
+        <span>{{hand}}</span> 
+     """.trimIndent()
+
+    override fun populateModel(slotName: String, model: Map<String, Any>): Map<String, Any> {
+        val desc = hand.joinToString(separator = ":", transform = {
+            card -> Card.cardDesc(card.value.toInt())
+        })
+        return model + mapOf("hand" to desc)
+    }
+
+    override fun onHandleUpdate(handle: Handle) {
+        // We only respond to changes to nextCard.
+        if (handle.name != "nextCard") return
+        val nc = nextCard.get()
+        if (nc == null) { return }
```

Comment:
You can use the elvis operator or a double bang here. Since you know nextCard was updated, and when it's updated it should have a value, either of these options should work.

Reply:
I did start with an elvis operator, but somehow ended up with this after refactoring. I will change it back. :)

Reply:
Nice. Done!